docs/31-long-running-job.md

```diff
@@ -0,0 +1,83 @@
+# Long Running Job Management
+
+The doAzureParallel package allows you to manage long running jobs easily. There are 2 ways to run a job:
+- Synchronous
+- Asynchronous
+
+Long running job should run in asynchronous mode.
+
+## How to configure a job to run asynchronously
+You can configure a job to run asynchronously by specifying wait = FALSE in job options:
+
+```R
+  options <- list(wait = FALSE)
```

Comment:
Add docs that you can also change the job id?

Reply:
fixed.